form/bootstrap5.rst

```diff
@@ -0,0 +1,263 @@
+            ],
+
+            // ...
+        ]);
+
+If you prefer to apply the Bootstrap styles on a form to form basis, include the
+``form_theme`` tag in the templates where those forms are used:
+
+.. code-block:: html+twig
+
+    {# ... #}
+    {# this tag only applies to the forms defined in this template #}
+    {% form_theme form 'bootstrap_5_layout.html.twig' %}
+
+    {% block body %}
+        <h1>User Sign Up:</h1>
+        {{ form(form) }}
+    {% endblock %}
+
+.. _reference-forms-bootstrap-error-messages:
+
+.. note::
+
+    By default, all inputs are rendered with the ``mb-3`` class on their container.
+    If you override the `row_attr` class option, the ``mb-3`` will be override too
+    and you will need to explicitly add it.
+
+Error Messages
+--------------
+
+Unlike the Bootstrap 4 template, errors are rendered **after** the ``input`` element.
+However, this still make a strong connection between the error and its ``<input>``, as
+required by the `WCAG 2.0 standard`_.
+
+Checkboxes and Radios
+---------------------
+
+For a checkbox/radio field, calling ``form_label()`` doesn't render anything.
+Due to Bootstrap internals, the label is already rendered by ``form_widget()``.
+
+Inline Checkboxes and Radios
+----------------------------
+
+If you want to render your checkboxes or radios fields `inline`_, you can add
+the `checkbox-inline` or `radio-inline` class, depending of your Symfony Form
+type or `ChoiceType` configuration, to the label class.
+
+.. configuration-block::
+
+    .. code-block:: php
+
+        $builder
+            ->add('myCheckbox', CheckboxType::class, [
+                'label_attr' => [
+                    'class' => '`checkbox-inline',
+                ],
+            ])
+            ->add('myRadio', RadioType::class, [
+                'label_attr' => [
+                    'class' => 'radio-inline',
+                ],
+            ]);
+
+    .. code-block:: twig
+
+        {{ form_row(form.myCheckbox, {
+            label_attr: {
+                class: 'checkbox-inline'
+            }
+        }) }}
+
+        {{ form_row(form.myRadio, {
+            label_attr: {
+                class: 'radio-inline'
+            }
+        }) }}
+
+Switches
+________
+
+Bootstrap 5 allows to render checkboxes as "`switches`_". You can enable this feature
+on your Symfony Form ``CheckboxType`` by adding the ``checkbox-switch`` class to the label:
+
+.. configuration-block::
+
+    .. code-block:: php
+
+        $builder->add('myCheckbox', CheckboxType::class, [
+            'label_attr' => [
+                'class' => 'checkbox-switch',
+            ],
+        ]);
+
+    .. code-block:: twig
+
+        {{ form_row(form.myCheckbox, {
+            label_attr: {
+                class: 'checkbox-switch'
+            }
+        }) }}
+
+.. tip::
+
+    You can also render your switches inline by simply adding the ``checkbox-inline`` class
+    on the ``label_attr`` option.
+
+    .. code-block:: php
```

Comment:
```suggestion
    on the ``label_attr`` option::
```

Reply:
@OskarStark Thanks for the solution, I had trouble understanding the error